commons-db/src/main/java/com/palantir/nexus/db/sql/SQLString.java

```diff
@@ -40,7 +40,9 @@
 @SuppressWarnings({"WeakerAccess", "BadAssert"}) // performance sensitive asserts
 public class SQLString extends BasicSQLString {
     private static final Pattern ALL_WORD_CHARS_REGEX = Pattern.compile("^[a-zA-Z_0-9\\.\\-]*$"); // $NON-NLS-1$
-    private static final String UNREGISTERED_SQL_COMMENT = "/* UnregisteredSQLString */";
+    private static final String UNREGISTERED_SQL_STRING = "UnregisteredSQLString"; // $NON-NLS-1$
+    private static final String UNREGISTERED_SQL_COMMENT =
+            "/* " + UNREGISTERED_SQL_STRING + " */"; // $NON-NLS-1$  // $NON-NLS-2$
```

Comment:
what does that comment represent?

Reply:
`/* UnregisteredSQLString */` is a SQL comment prefixed on statements that helps to identify unregistered (often runtime dynamically generated) SQL that is being executed compared to registered (static SQL structures). Some applications use registered SQL statements to swap in alternative SQL hints to adjust the query execution plans at runtime to work around cases where a database chooses an inefficient plan. In many cases, this functionality has been incorporated into databases themselves (e.g. Oracle added Adaptive Plans in 12c) so this is less important these days, but provided for backward compatibility.